networkx/algorithms/tests/test_covering.py

```diff
@@ -0,0 +1,89 @@
+from nose.tools import assert_equal, assert_true
+import networkx as nx
+
+r"""Tests for :func:`min_edge_cover`"""
```

Comment:
Raw docstrings in the middle of a module are not good. Instead, collect related unit tests into a single test class, for example `class TestMinEdgeCover`; then you can just provide a docstring for the class. Also you can remove the `min_edge_cover_` prefix from all of the test methods in that class.